### PR TITLE
Adding guide 

### PR DESCRIPTION
**Pull request by justinbmeyer**
for #22

### CODE REVIEW

**Comment by justinbmeyer** [comment]
We might want to add a section about testing with dependency injection if  #9 lands.

File: guide.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ 	- Why it exists.
+ 	- High level functionality.
+ 		- Include ability to login/out and functionality 
+ 		  dynamically changes.
+ 	- Where to use it. (link to actual app)
+ 	- What you are going to learn.
+ - High level architecture.
+ 	- Characteristics: 
+ 		- Express/NodeJS + DoneJS + Postgres/Bookshelf
+ 		- Deployed to heroku + firebase
```
Comment (akagomez) [inline guide.md]: Does Firebase do Postgres hosting?

Comment (justinbmeyer) [inline guide.md]: Heroku does.

Comment (akagomez) [inline guide.md]: I guess I'm confused then. What are we deploying to Firebase?

Comment (justinbmeyer) [inline guide.md]: static assets: https://donejs.com/Guide.html#section=section_Deploy

**Pull request merged**